moveit_py/src/moveit/moveit_core/planning_scene/planning_scene.cpp

```diff
@@ -181,15 +181,15 @@ void init_planning_scene(py::module& m)
                color (moveit_msgs.msg.ObjectColor, optional): The color of the collision object. Defaults to None if not specified.
            )")
 
-      .def("set_object_color", &planning_scene::PlanningScene::setObjectColor, py::arg("object_id"),
-           py::arg("color_msg"),
-           R"(
-           Set the color of a collision object.
+      //  .def("set_object_color", &planning_scene::PlanningScene::setObjectColor, py::arg("object_id"),
+      //       py::arg("color_msg"),
+      //       R"(
+      //       Set the color of a collision object.
 
-	   Args:
-               object_id (str): The id of the collision object to set the color of.
-               color (std_msgs.msg.ObjectColor): The color of the collision object.
-           )")
+      //    Args:
+      //           object_id (str): The id of the collision object to set the color of.
+      //           color (std_msgs.msg.ObjectColor): The color of the collision object.
+      //       )")
```

Comment:
Hi @AndyZe, 

Thanks for notifying me of this. For now it would make sense to comment out this code and add a todo item. In order to account for these changes I would need to add bindings for the new class you introduced or reference the `object_colors_` from the planning scene instance. I think this is relatively low priority for the Python library right now so I would be happy to just go with a todo item in the code that can be addressed if/when it becomes a priority.

1. Bind the new class PlanningSceneColors
2. Refer to object colors using `object_colors_` from the planning scene